### PR TITLE
Created a new file called _targets.R

### PR DESCRIPTION
**Pull request by algorsky**
Added the file _targets.R to a makefile.

### CODE REVIEW

**Comment by algorsky** [comment]
`> targets::tar_make()
✓ skip target model_RMSEs_csv
✓ skip target eval_data
✓ skip target model_summary_results_csv
• start target Temp_RMSEs_png
• built target Temp_RMSEs_png
✓ skip target model_diagnostic_text_txt
• end pipeline
 `

**Comment by lindsayplatt** [comment]
I think if you use three backticks, it will format that as a block of code instead of just a line of code. Just so you know!

File: _targets.R [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    eval_data,
+    add_colors_to_data(in_filepath = model_RMSEs_csv),
+  ),
+  # Create a plot
+  tar_target(
+    Temp_RMSEs_png,
+    plot_data(file_name = 'Temp_RMSEs.png', 
+              project_directory = '3_visualize/out/', 
+              eval_data = eval_data,
+              file_out = '3_visualize/out/Temp_RMSEs.png'),
```
Comment (lindsayplatt) [inline _targets.R:26]: If your `file_out` has `3_visualize/out` in it, then I don't think you also need `project_directory`.

Comment (lindsayplatt) [reply _targets.R:26]: You also have `file_name` specified, but I think you were using that for the input file. I think you can delete some of the arguments.

File: _targets.R [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+              file_out = '3_visualize/out/Temp_RMSEs.png'), 
+    format = "file"
+  ),
+  # Save the processed data
+  tar_target(
+    model_summary_results_csv,
+    save_csv(data_in = eval_data, 
+              project_directory = '2_process/out/',
+              file_name = 'model_summary_results.csv',
+             file_out = '2_process/out/model_summary_results.csv'),
```
Comment (lindsayplatt) [inline _targets.R]: Same thing here - you don't need both `file_name` + `project_directory` and then also `file_out`. Just one of them and then use the one that you want to inside the function.

File: _targets.R [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+             file_out = '2_process/out/model_summary_results.csv'), 
+    format = "file"
+  ),
+  # Save the model diagnostics
+  tar_target(
+    model_diagnostic_text_txt,
+    generate_model_diagnostics(data_in = eval_data,
+                               project_directory = '2_process/out/',
+                               file_name = "model_diagnostic_text.txt",
+                               file_out = '2_process/out/model_diagnostic_text.txt'),
```
Comment (lindsayplatt) [inline _targets.R:40]: Same again here with the duplicate file name inputs!

**Review by lindsayplatt — APPROVED** [review]
You just have a few instances where you are passing in duplicate information to the function. Once you fix those, good to go and merge!

**Pull request merged**